src/screens/FeedScreen/Content.js

```diff
@@ -363,7 +377,6 @@ export const styles = StyleSheet.create({
     lineHeight: 24,
     flex: 1
   },
-
   seemore: {
     color: COLORS.blue
   },
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The `lineHeight` property has been added to the `text` style. Make sure this does not affect the layout of your text elements in an undesired way.
```diff
+ lineHeight: 24,
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->